tutorials/aframe-tool-tutorial.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: doc
+title: Creating A-Frame Tools
+permalink: /docs/tutorials/aframe-tool-tutorial
+---
+
+## Intermediate Tool Tutorial: 3D Graphics with A-Frame
+
+Pre-reqs: [Creating a Simple Tool](../develop/spatial-tools/tutorial) and
+[How to Create an Object](../use/connect-to-the-physical-world/create-object)
+
+In this tutorial we'll take a basic a-frame scene and turn it into a spatial
+tool with a couple minor modifications. [A-Frame]() is a framework that allows
+us to write simple HTML to generate 3D scenes.
+
+
+<video controls width="640">
+    <source src="images/aframe-tool/aframe-demo.mov"
+            type="video/mov">
+
+    <source src="images/aframe-tool/aframe-demo.webm"
+            type="video/webm">
+
+    Sorry, your browser doesn't support embedded videos.
+</video>
```

Comment:
Cool video! I wasn't sure if they worked so I've been using gifs, but I may try this format in the future

Reply:
Yep! I was being lazy and didn't want to find the correct parameters to get a gif together but video is better in terms of quality per mb anyway.